lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -196,6 +197,71 @@ def test_create_ccx(self):
         self.assertEqual(response.status_code, 200)
         self.assertTrue(re.search('id="ccx-schedule"', response.content))
 
+    @SharedModuleStoreTestCase.modifies_courseware
+    @patch('ccx.views.render_to_response', intercept_renderer)
+    @patch('ccx.views.TODAY')
+    def test_get_ccx_schedule(self, today):
```

Comment:
@giocalitri I think we need tests for scenarios:

- When chapter is hidden for student but section and subsections are not.
- When subsections or single subsection hidden from students but chapter and units are not
- When one or all units are hidden from students but chapter and subsections are not.